examples/index.md

```diff
@@ -162,3 +162,241 @@ Content-Type: application/vnd.api+json
+while creating/updating the resource, and that this error was caused
+by an invalid `"first-name"` attribute:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "status": "422",
+      "source": { "pointer": "/data/attributes/first-name" },
+      "title":  "Invalid Attribute",
+      "detail": "First name must contain at least three characters."
+    }
+  ]
+}
+```
+
+Every member in an error object is optional, but all help the client
+by providing extra details.
+
+The `source` member is used to [indicate](https://tools.ietf.org/html/rfc6901)
+which part of the request document caused the error.
+
+The `title` and `detail` members are similar, but `detail` is specific
+to this occurrence of the problem, whereas `title` is more generic.
+
+The `status` member represents the HTTP status code associated with the problem.
+It's very helpful when multiple errors are returned at once (see below), as the
+HTTP response itself can only have one status code. However, it can also be
+useful for single errors, to save clients the trouble of consulting the HTTP
+headers, or for using JSON API over non-HTTP protocols, which may be officially
+supported in the near future.
+
+### Multiple Errors
+
+When multiple errors occur in response to a single request, the server
+can simply add each error to the `errors` array:
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "status": "403",
+      "source": { "pointer": "data/attributes/secret-powers" },
+      "detail": "Editing secret powers is not authorized on Sundays."
+    },
+    {
+      "status": "422",
+      "source": { "pointer": "data/attributes/volume" },
+      "detail": "Volume does not, in fact, go to 11."
+    },
+    {
+      "status": "500",
+      "source": { "pointer": "data/attributes/reputation" },
+      "title": "The backend responded with an error",
+      "detail": "Reputation service not responding after three requests."
+    }
+  ]
+}
+```
+
+The only uniqueness constraint on error objects is the `id` field. Thus,
+multiple errors on the same attribute can each be given their own error
+object. The example below shows multiple errors on the `first-name` attribute:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
+      "detail": "First name must contain an emoji."
+    }
+  ]
+}
+```
+
+> Note: in the responses above with a 422 status code, `400 Bad Request` would
+also be acceptable. ([http://stackoverflow.com/a/20215807/1261879](More details.))
+JSON API doesn't take a position on 400 vs. 422.
```

Comment:
This is there to mark the section as an indented note. Cf the note above that says "The above example URI shows unencoded [ and ]..."